docs/portal/transports/add-transport.md

```diff
@@ -20,15 +20,15 @@ The <strong> Transports </strong> page can be reached simply by clicking "Transp
 | Fields               | Description   |
 | -------------------- | ------------- |
 | Fixed Shields        | The shield servers (the HDT servers that connection to your origin servers) will be fixed. |
-| Dedicated IP Service | The IPs for serving the transport will be dedicated, and only for the cname. The option will be NOT applicable if you select HTTP or HTTPs as the Application Protocol. If you wish to use <strong> Dedicated IP Service </strong> for your HTTP/HTTPs applications, please select Others as the Application Protocol.|
+| Dedicated IP Service | The IP addresses for serving the transport will be dedicated, and only for the cname. The option will NOT be applicable if you select HTTP or HTTPs as the Application Protocol. To use <strong> Dedicated IP Service </strong> for your HTTP/HTTPs applications, select Others as the Application Protocol.|
```

Comment:
Please consider changing the first sentence to: "The IP addresses for serving the transport is dedicated to the cname only."
Should "cname" be capitalized here and wherever else it is used?
In the second sentence, please change the capitalization from "HTTPs" to "HTTPS."